content/open-source/oci/what-are-oci-artifacts.md

```diff
@@ -44,9 +44,9 @@ In the example manifest above, the  `config` field contains the custom `mediaTyp
 Some container tools make use of the OCI Artifacts format guidelines (such as Helm and Tekton), but using these guidelines comes with a serious drawback. Not all registries support OCI Artifacts (or manifests with a custom `mediaType`), and the [OCI Image Specification](https://github.com/opencontainers/image-spec) recommends avoiding the use of Artifacts if you are concerned about portability. As you will read about in the section below, this lack of portability is a reason why some software projects choose to store artifacts in an OCI registry without adding a custom `mediaType` to the manifest.
 
 ## OCI artifacts and software supply chain security
-For software supply chain security, OCI artifacts offer a useful way to store [SBOMs](​​https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) and signatures inside a container registry.
+For software supply chain security, OCI artifacts offer a useful way to store [SBOMs](/open-source/sbom/what-is-an-sbom/) and signatures inside a container registry.
```

Comment:
This URL had two `\u200b` (non-printing space) characters at the beginning that may or may not have been messing with rendered links.